elasticms-admin/config/packages/security.yaml

```diff
@@ -85,7 +85,6 @@ security:
         - { path: ^/api, roles: ROLE_API }
         - { path: ^/api/admin, roles: ROLE_ADMIN }
         - { path: ^/admin, roles: ROLE_ADMIN }
-        - { path: ^/i18n-type, roles: ROLE_ADMIN }
         - { path: ^/action, roles: ROLE_ADMIN }
         - { path: ^/wysiwyg-profile, roles: ROLE_ADMIN }
```

Comment:
This should be removed also. Maybe there is other rules to clean?

Reply:
https://github.com/ems-project/elasticms/pull/974/commits/cba7cded1d2a4f2b2a490fb968a60c2250dcbdb5

In feature pr's maybe these should also be prefixed with just /admin and removed from security.yaml

```yaml
        - { path: ^/action, roles: ROLE_ADMIN }
        - { path: ^/content-type, roles: ROLE_ADMIN }
        - { path: ^/environment, roles: ROLE_ADMIN }
        - { path: ^/channel-admin, roles: ROLE_ADMIN }

´´´´